skimage/segmentation/tests/test_active_contour_model.py

```diff
@@ -118,29 +117,3 @@ def test_bad_input():
         active_contour(img, init, max_iterations=-15,
                        coordinates='rc')
 
-
-def test_bc_deprecation():
-    with expected_warnings(['boundary_condition']):
-        img = rgb2gray(data.astronaut())
-        s = np.linspace(0, 2*np.pi, 400)
-        r = 100 + 100*np.sin(s)
-        c = 220 + 100*np.cos(s)
-        init = np.array([r, c]).T
-        snake = active_contour(gaussian(img, 3), init,
-                               bc='periodic', alpha=0.015, beta=10,
-                               w_line=0, w_edge=1, gamma=0.001,
-                               max_iterations=100, coordinates='rc')
-
-
-def test_xy_coord_warning():
-    # this should raise ValueError after 0.18.
```

Comment:
@sciunto as per this comment, this test should be brought back with `with testing.raises(ValueError)`